src/components/pages/pages.pw.tsx

```diff
@@ -0,0 +1,194 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+
+import { PagesProps } from "../pages";
+import {
+  dataComponentButtonByText,
+  backArrow,
+} from "../../../playwright/components/pages";
+import {
+  getComponent,
+  getDataElementByValue,
+} from "../../../playwright/components";
+import { CHARACTERS } from "../../../playwright/support/constants";
+import {
+  checkAccessibility,
+  waitForAnimationEnd,
+} from "../../../playwright/support/helper";
+import {
+  PageComponent,
+  PageComponentWithTitle,
+  PagesComponent,
+} from "./components.test-pw";
+
+test.describe("Prop checks for Pages component", () => {
+  ["initialPageIndex", "pageIndex"].forEach((propName) => {
+    test.describe(`when ${propName}`, () => {
+      [
+        ["number", 1],
+        ["string", "1"],
+      ].forEach(([type, propValue]) => {
+        test(`is passed as ${type}, should show the title that matches the one on the page with that index`, async ({
+          mount,
+          page,
+        }) => {
+          const props = { [propName]: propValue };
+
+          await mount(<PagesComponent {...props} />);
+
+          await dataComponentButtonByText(page, "Open Preview").click();
+
+          await expect(getDataElementByValue(page, "title")).toHaveText(
+            "My Second Page"
+          );
+        });
+      });
+    });
+  });
+
+  // TODO: Playwright is not returning the full class attribute, so this test is not working as expected.
+  (["slide", "fade"] as PagesProps["transition"][]).forEach((transition) => {
```

Comment:
**comment** - we shouldn't skip tests unless there's something making it fail that defies easy explanation.

In this case I think the issue is that we're not actually clicking the "next page" button, so the browser never actually moves on to the next page and so no animations or related CSS classes happen.